### ISSUE TITLE
map_to_weighting may fail if sf is tibble

### ISSUE THREAD

**Issue by alinekirkland** [mabafaba/hypegrammaR#16]

**Comment by mabafaba** [comment]
resolved (in mabafaba/surveyweights) 24952a325efa9362f23a79c0a0b2e01b196c9a7a

**Comment by zackarno** [comment]
tibble still causes map_to_weighting to fail in the version I have?

**Comment by mabafaba** [comment]
Thanks. Might have forgotten to push - will check

**Comment by ellieallien** [comment]
Tibble still causing issues for @yannsay as well

**Comment by mabafaba** [comment]
is this still happening?

**Comment by yannsay** [comment]
yes, this is still happening with tibble. it works fine with a data frame.
